tests/Application/ConfigResolverTest.php

```diff
@@ -123,16 +126,16 @@ public function testResolvedConfigIsCorrectlyMerged(): void
         );
     }
 
-    public function testUnknownPresetThrowException(): void
+    public function testUnknownPresetReturnsDefault(): void
     {
-        $this->expectException(InvalidPreset::class);
```

Comment:
:thinking: I'm not sure if we should drop that checking. 
That let's an user to have an invalid preset in configuration and we'll not advert him that the configuration is incorrect... 

@olivernybroe any advice here ?

Reply:
Check out https://github.com/nunomaduro/phpinsights/pull/440#issuecomment-727265706 and let me know if that works for you.